packages/office-ui-fabric-react/src/components/Nav/Nav.Props.ts

```diff
@@ -173,6 +183,12 @@ export interface INavLink {
   parentId?: string;
 
   /**
+   * (Optional) By default, any link with onClick defined will render as a button.
+   * Set this property to true to override that behavior.
+   */
+  isAnchor?: boolean;
```

Comment:
since majority would be isAnchor true, should we do isButton so by default everything is anchor unless isButton is set to true that is smaller case?  unless you default isAnchor to true otherwise too many place to set?